src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -93,6 +93,21 @@ public User getUser(UUID id) {
     return null;
   }
 
+  /** Find and return User list. */
```

Comment:
Please add unit tests for these new methods

Reply:
Unit tests for these methods have been written.